src/openrct2/object/ObjectManager.cpp

```diff
@@ -374,20 +381,19 @@ class ObjectManager final : public IObjectManager
         }
         if (slot.has_value())
         {
-            auto* object = GetOrLoadObject(ori);
+            auto object = GetOrLoadObject(ori);
             if (object != nullptr)
             {
                 if (_loadedObjects.size() <= static_cast<size_t>(*slot))
                 {
                     _loadedObjects.resize(slot.value() + 1);
                 }
                 loadedObject = object;
-                _loadedObjects[slot.value()] = object;
+                _loadedObjects[slot.value()] = std::move(object);
```

Comment:
Is this change correct, @Zehmatt ? I know some stuff has changed in develop since this was put in, so it may be wrong/outdated.

Reply:
Yeah its probably not right, GetOrLoadObject returns a pointer value now.